manifests/sentinel.pp

```diff
@@ -200,9 +200,13 @@
 
   if $::osfamily == 'Debian' {
     # Debian flavour machines have a dedicated redis-sentinel package
-    # This is default in Xenial onwards, unstable debian or PPA/other upstream
+    # This is default in Xenial or Stretch onwards or PPA/other upstream
     # See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=775414 for context
-    if (versioncmp($::operatingsystemmajrelease, '16.04') >= 0 or $::redis::manage_repo) {
+    if (
+      versioncmp($::operatingsystemmajrelease, '16.04') >= 0 or
```

Comment:
This should also include something like `$::operatingsystem == "ubuntu"` and below the same for `"debian"`.

Reply:
... and still I think the condition should go to params.pp. This way it is still broken for debian-jessie and older, as the `require => Package[$package_name];` some lines later will fail otherwise.

Reply:
Good point, the edgecase is unlikely (Ubuntu on 9 or earlier) but might as well protect against it

Reply:
The require package name works fine, because on Jessie the package is redis instead of redis-sentinel. The tests work, both unit and acceptance 👍